Add 20230717 work on graphs, maps, Jupityr notebook

m-ball3 opened the pull request.

m-ball3 committed:
Add 20230717 work on graphs, maps, Jupityr notebook

leewujung reviewed (commented):
Great work! A few small comments:
- the python scripts use the extension `.py` instead of `.txt` -- this way the IDE (like pycharm, VScode) would recognize the type of text file that is
- for temperature, even though the color code does look VERY nice, it seems to make the transparency a little difficult to see? Maybe it's a matter of setting the alpha value smaller
- the plots if you look closely the marker edges are plotted without an alpha value. You could do `markeredgecolor=None` to simply remove the edges.

leewujung commented:
Also, there are some merge conflicts, so you need to resolve them by merging `main` into this branch. I would mostly be in meetings tomorrow but @aditya-uw could you please help @m-ball3 along with when you guys are going over the conda environment setup. Thanks!

m-ball3 commented:
Here is an example of my graphs with the alpha function! Opacity was set to .1 because higher opacity was hard or impossible to see. I didn't think that that feature quite conveyed what I thought it would, but am happy to make the change (I can easily add the trendlines). 

![image](https://github.com/uw-echospace/batmo/assets/136123608/433f37e2-206b-440b-8534-4a0eee87c46c)

![image](https://github.com/uw-echospace/batmo/assets/136123608/9352aa43-a286-4153-8717-c6de110c5aea)

m-ball3 committed:
Fix conflict

Update local so that local and main match

m-ball3 committed:
Fix conflict

Update local meteorological-data-source.md so that local and main match

m-ball3 committed:
Merge branch 'main' into metdata-graphs-20230717

m-ball3 committed:
Update .txt files to .py files

Update graph codes and import codes from .txt to .py per Wu-Jung's notes. Allow Code Highlighting.

m-ball3 committed:
Add local changes to Main

m-ball3 committed:
Merge remote-tracking branch 'origin/metdata-graphs-20230717' into metdata-graphs-20230717
